WormsApp/MapperWrapper.h

```diff
@@ -0,0 +1,18 @@
+#ifndef MAPPERWRAPPER_H
+#define MAPPERWRAPPER_H
+
+#include <unordered_map>
+#include <utility>
+#include<thread>
+
+template <typename KeyType, typename valueType> class MapperWrapper
```

Comment:
nit: names should follow one style - `KeyType` and `ValueType` or alternatively `keyType` and `valueType`.